skimage/color/colorconv.py

```diff
@@ -637,9 +637,7 @@ def xyz2rgb(xyz):
     mask = arr > 0.0031308
     arr[mask] = 1.055 * np.power(arr[mask], 1 / 2.4) - 0.055
     arr[~mask] *= 12.92
-    arr[arr < 0] = 0
-    arr[arr > 1] = 1
-    return arr
+    return np.clip(arr, 0, 1)
```

Comment:
you should at least clip in place. Please use a kwarg for that parameter.

`a_min` and `a_max` may be used at your discretion.